ElectronClient/app/gui/note-viewer/index.html

```diff
@@ -59,7 +59,7 @@
 	let hljsScriptAdded = false;
 	let hljsLoaded = false;
 
-	function loadHljs(callback) {
+	function loadHljs(codetheme) {
```

Comment:
Please could you call the argument an "options" object with (for now) just one property "codetheme"? That way it will be possible to add more options later on, which is easier than adding more function arguments.

Reply:
i.e. `function loadHljs(options)` and below, of course, `link.href = 'highlight/styles/' + options.codetheme;`